### DISCUSSION TITLE
Which mariadb or mysql version is recommended for ejbca 7.10.2?

### DISCUSSION THREAD

**Discussion by kvd-admin** [Keyfactor/ejbca-ce#296] [General]
Hi together,

i found a very good EJBCA Compatibility Guide for version 6.15 -> https://doc.primekey.com/ejbca6152/ejbca-installations/ejbca-main-installation/ejbca-compatibility-guide ... but there is no guide for Version 7.10.

So, which mariadb or mysql version is recommended for ejbca 7.10.2?

Thanks
Markus

**Comment by primetomas** [comment]
We quickly figured out that database (or any software) versions move too fast nowadays to keep a matrix of any minor versions.
The latest version is recommended.
